root/brent_test.go

```diff
@@ -0,0 +1,405 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package root_test
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"gonum.org/v1/exp/root"
+	"gonum.org/v1/gonum/floats/scalar"
+)
+
+var eps = math.Nextafter(1, 2) - 1
+
+func TestBrent(t *testing.T) {
+	t.Parallel()
+
+	type testcase struct {
+		name string
+		f    func(float64) float64
+		a    float64
+		b    float64
+		tol  float64
+		want float64
+	}
+	var tests []testcase
+	for _, f := range tstutilsFns {
+		tests = append(tests, testcase{name: f.name, f: f.f, a: .5, b: math.Sqrt(3), tol: 4 * eps, want: 1})
```

Comment:
```suggestion
		tests = append(tests, testcase{name: f.name, f: f.f, a: 0.5, b: math.Sqrt(3), tol: 4 * eps, want: 1})
```
(a single dot is too little syntax)